homeassistant/components/light/hue.py

```diff
@@ -285,6 +285,25 @@ def effect_list(self):
         """Return the list of supported effects."""
         return [EFFECT_COLORLOOP, EFFECT_RANDOM]
 
+    @property
+    def device_info(self):
+        """Return the device info."""
+        if self.light.type in ('LightGroup', 'Room'):
+            return None
+
+        return {
+            'identifiers': {
+                (hue.DOMAIN, self.unique_id)
+            },
+            'name': self.name,
+            'manufacturer': self.light.manufacturername,
+            # productname added in Hue Bridge API 1.24
+            # (published 03/05/2018)
+            'model': self.light.productname or self.light.modelid,
+            # Not yet exposed as properties in aiohue
```

Comment:
Created a PR so in the future we can replace this with property calls: https://github.com/balloob/aiohue/pull/10